src/c_api.rs

```diff
@@ -195,46 +200,40 @@ pub extern "C" fn JSONAPI_getStringFromKey(
         let key = unsafe { &mut *key };
         let path = unsafe { CStr::from_ptr(path).to_str().unwrap() };
         if let Ok(s) = key.redis_json.to_string(path, Format::JSON) {
```

Comment:
This should be similar to `JSONAPI_getString` (that is, no `.to_string(path, Format::JSON)`.